native/navigation/route-names.js

```diff
@@ -30,6 +30,8 @@ export const MoreRouteName = 'More';
 export const MoreScreenRouteName = 'MoreScreen';
 export const ChatRouteName = 'Chat';
 export const ChatThreadListRouteName = 'ChatThreadList';
+export const HomeChatThreadListRouteName = 'Home';
+export const BackgroundChatThreadListRouteName = 'Background';
```

Comment:
You'll need to create a new `ParamList` for your new navigator in this file, and make sure it's included in the global `ScreenParamList`